src/main/java/codeu/model/data/User.java

```diff
@@ -79,6 +65,11 @@ public Instant getCreationTime() {
     return creation;
   }
 
+  /** Returns whether User is admin or not. */
+  public Boolean getIsAdmin() {
```

Comment:
Might it be a good idea to have a setter too? What if we a current user to become an admin?